roles/ansible-deploy-rudder/library/rudder_server_settings.py

```diff
@@ -85,7 +83,9 @@ class RudderSettingsInterface(object):
     def __init__(self, module):
         self._module = module
         # {{{ Authentication header
-        self.headers = {"Content-Type": "application/json"}
+        self.headers = {
+            "Content-Type": "application/json"
```

Comment:
you need to concatenate all redefinition of self.headers with this one, or force the application/json in every others usage (like l91)

Reply:
and l95

Reply:
This is now fixed, thanks for the feedback, it should fix the problem of sending the data.

Reply:
thank you !